README.md

```diff
@@ -377,29 +391,11 @@ command args ...
 
 since bats sets `set -e` for all tests.
 
-2. In case you want to hide the command output (which `run` does), use output redirection instead.
-
-This
-
-```bash
-run command ...
-[ "$status" -eq 0 ]
-```
-
-is equivalent to
-
-```bash
-command ... >/dev/null
-```
-
-Note that the output is only shown if the test case fails.
-
-3. In case you need to assign command output to a variable (and maybe check
+2. In case you need to assign command output to a variable (and maybe check
```

Comment:
I think this whole "when not to use run" section needs to be renamed rewritten, now that we actually want to support these cases directly with run.

Reply:
Hmmm, agreed. I thought I had done a better job of editing that. Deferring for now, until we converge on something we're all happy with. (Note to self, I'd also like to add a warning about using redirection in `run`)